### ISSUE TITLE
Issue with FDS 6.8 linux binaries

### ISSUE THREAD

**Issue by johodges** [firemodels/fds#11764]
Tried to install the new release on a linux cluster and ran into an issue. The shell file shows up as having windows line endings even though I copied it directly from the repo with wget. I am able to run dos2unix on the file to change the line endings but then the script does not install all the files, probably due to dos2unix replacing some of the binary data. Running the same process within WSL on my local machine has the same result.

I am able to compile the packaged source files and run on our cluster, so it looks like it is an issue with the bash wrapper. Are others having the same issue?

**Comment by gforney** [comment]
Ill take a look.

**Comment by marcosvanella** [comment]
So, I added the source of FDSVARS.sh, SMVVARS.sh:

```
mnv@blaze mnv $ source /home/mnv/FDS/FDS6/bin/FDS6VARS.sh
mnv@blaze mnv $ source /home/mnv/FDS/FDS6/bin/SMV6VARS.sh
```

The go to the ~/FDS/FDS6/Examples/Pressure_Solver dir:

```
mnv@blaze Pressure_Solver $ which fds
~/FDS/FDS6/bin/fds
mnv@blaze Pressure_Solver $ which mpirun
~/FDS/FDS6/bin/INTEL/bin/mpirun
mnv@blaze Pressure_Solver $ mpirun -n 4 fds dancing_eddies_default.fds 
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found
MPI startup(): Warning: I_MPI_PMI_LIBRARY will be ignored since the hydra process manager was found

 Starting FDS ...

 MPI Process      0 started on blaze.el.nist.gov
 MPI Process      2 started on blaze.el.nist.gov
 MPI Process      1 started on blaze.el.nist.gov
 MPI Process      3 started on blaze.el.nist.gov

 Reading FDS input file ...


 Fire Dynamics Simulator

 Current Date     : April 18, 2023  17:35:07
 Revision         : FDS-6.8.0-0-g886e009-release
 Revision Date    : Tue Apr 18 07:06:40 2023 -0400
 Compiler         : ifort version 2021.7.1
 Compilation Date : Apr 18, 2023 15:20:17

 MPI Enabled;    Number of MPI Processes:       4
 OpenMP Disabled

 MPI version: 3.1
 MPI library version: Intel(R) MPI Library 2021.6 for Linux* OS


 Job TITLE        : Simple 2D Tunnel
 Job ID string    : dancing_eddies_default

 Time Step:      1, Simulation Time:     0.004 s
 Time Step:      2, Simulation Time:     0.008 s
 Time Step:      3, Simulation Time:      0.01 s
 Time Step:      4, Simulation Time:      0.02 s
 Time Step:      5, Simulation Time:      0.02 s


```

What files are you missing?

**Comment by johodges** [comment]
Here is what I am seeing. The root cause of the issue is having to change the line endings which is affecting the binary data as well. I can probably work around it by just replacing the line endings in the first N lines, but it is odd that I don't see this behavior on other releases.
![00_wget_permissions](https://user-images.githubusercontent.com/17127980/232915629-ecf4562e-3f86-4eee-83f5-30b8d70007a2.PNG)
![01_initial_bash_run](https://user-images.githubusercontent.com/17127980/232915631-302e8595-c177-4638-a7c1-bebc61542411.PNG)
![02_lineending_replace_second_bash_run](https://user-images.githubusercontent.com/17127980/232915632-b9c58e29-e946-40b9-8ea1-672829f3f435.PNG)
![03_allsavedfiles](https://user-images.githubusercontent.com/17127980/232915637-72bebe26-35a4-4d06-a836-1b79c28202ee.PNG)

**Comment by marcosvanella** [comment]
It is interesting I don't pick it up on our Linux distribution. We are using Centos 7. 
We'll let Glenn have a look at the script.

**Comment by gforney** [comment]
I was seeing the dos line endings too.

try again.  I was tranferring the bundles from one github release to another.  I downloaded to the PC then uploaded from the PC.  I think pscp (PC equivalent of the ssh program scp) was fooled and thought the .sh files were text files and added the dos line endings.  anyway, I uploaded the buundles directly from blaze - PC was not in the picture.  I'm now not seeing the dos line endings.
